### PR TITLE
[Fix Docs] Improve code formatting

### LINKED ISSUE #9897: [Docs] Improve code formatting
### Current State
On https://docs.meshery.io/installation/docker/docker-extension page, the current state of code snippets: 
Screenshot from 2024-01-09 18-59-30

### Desired State
format it to look like:
Screenshot from 2024-01-09 19-04-34

Note: Don't modify the actual content.

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by sandramsc**
Removed input text "mesheryctl system dashboard"

This PR fixes #9897 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 614875993c52198dbb25a16903eb461df164b437 at: https://659ee81bfd23e814effba654--meshery-docs-preview.netlify.app

**Review by vishalvivekm — CHANGES_REQUESTED** [review]
@sandramsc I think I wasn't clearly descriptive of rhe ask here, but please find the note in the Issue description which says, don't modify the actual content ( commnds/code snippets ).

The ask is to remove the extra whitespace from the terminal, which finally will look like similar to the desired screenshot.

**Comment by sandramsc** [comment]
> @sandramsc I think I wasn't clearly descriptive of rhe ask here, but please find the note in the Issue description which says, don't modify the actual content ( commnds/code snippets ).
> 
> The ask is to remove the extra whitespace from the terminal, which finally will look like similar to the desired screenshot.

Thanks for clarifying as the current state screenshot did include the text **mesheryctl system dashboard**, however, the desired screenshot did not include this text. Will update the PR with the feedback.

**Comment by l5io** [comment]
🚀 Preview for commit 4611d7c7c25f513b6a8ba86043c968b612470eec at: https://65a02c69bec86d194bb524c4--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 3996a2c4143bb6ce33cf4bd4c18ef9cb7fabfd25 at: https://65a034de54fcee1f541a531c--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 3fdb29f6d3a074f227428df3c7209a198031c577 at: https://65a0411c511fda2a7d8613d0--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 705aa4852879ce9c3cda6d06e00ddc2fabc342bd at: https://65a0431f511fda2ccb860c99--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 47c38a32c451c3ec06ebc272f8661b3bc69d35de at: https://65a2a2a33a1518d13959822b--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 97f621ebe8c5cb64a8b458f80a009283940d6d87 at: https://65a2b74999d609dcb9b26d1f--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit ccc51cf84b4d2a9efb0537f4dac7ac8166baf505 at: https://65a2be5685c506dc5236d926--meshery-docs-preview.netlify.app

File: docs/pages/installation/docker-extension.md [hunk 1]
```diff
@@ -100,7 +103,7 @@ If you want to remove Meshery as a Docker extension from your system, follow the
    - Meshery might have pulled Docker images for its components. You can remove these images using the `docker rmi` command. Replace the image names with the actual ones you want to remove:
 
    <pre class="codeblock-pre" style="padding: 0; margin-top: 2px; font-size:0px;"><div class="codeblock" style="display: block;">
-    <div class="clipboardjs" style="padding: 0">
+    <div class="clipboardjs" style="padding: 0; height: 8px; overflow: hidden;">
```
Comment (MUzairS15) [inline docs/pages/installation/docker-extension.md:106]: I know, the usage of `px` has been in practice for a long on these pages, but can we get rid of them as we add new styles?

Comment (leecalcote) [reply docs/pages/installation/docker-extension.md:106]: @sandramsc the implied suggestion here is to use `rem` as the unit of measure.

P.S. 

`0.5rem` is equivalent to `8px`

File: docs/pages/installation/docker-extension.md [hunk 2]
```diff
@@ -47,20 +47,23 @@ Meshery runs as a set of one or more containers inside your Docker Desktop virtu
...
+      <span style="font-size: 0;">docker extension install meshery/docker-extension-meshery</span> 
+    </div>
+    <div class="window-buttons"></div>
+    <div id="termynal2" style="width: 100%; height: 200px; max-width: 100%;" data-termynal="">
+      <span data-ty="input">docker extension install meshery/docker-extension-meshery</span>
+      <span data-ty="progress"></span>
+      <span data-ty="">Successfully installed Meshery</span>
+      <span data-ty="input">mesheryctl system dashboard</span>
+    </div>
```
Comment (captain-Akshay) [inline docs/pages/installation/docker-extension.md:62]: These are so good!

**Comment by l5io** [comment]
🚀 Preview for commit 6a3019ac6a42c52086bd332ac818f342cc5d4d3d at: https://65afd840701e0c23851d5b13--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 679d3cdf9a11bec473ef0dc6ea202cca2a32b0f7 at: https://65afdaa9deadc2254af63e97--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 0264de69964d6bf135faf7dfb1d16beb36b43ea3 at: https://65c002e09ae486aa392a6279--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
@vishalvivekm, please give this a final review before merging.

**Review by leecalcote — APPROVED** [review]
Thanks, @sandramsc! 👏 🙂

**Pull request merged**